src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -106,9 +107,10 @@ private static Person createEditedPerson(Person personToEdit, EditPersonDescript
         Phone updatedPhone = editPersonDescriptor.getPhone().orElse(personToEdit.getPhone());
         Email updatedEmail = editPersonDescriptor.getEmail().orElse(personToEdit.getEmail());
         Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
+        Amount updatedAmount = editPersonDescriptor.getAmount().orElse(personToEdit.getAmount());
         Set<Tag> updatedTags = editPersonDescriptor.getTags().orElse(personToEdit.getTags());
 
-        return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedTags);
+        return new Person(updatedName, updatedPhone, updatedEmail, updatedAddress, updatedAmount, updatedTags);
```

Comment:
I think we should not be able to modify the amount of a person through EditCommand.
Just think of what would happen if you do that, how to calculate someone balance ?

Reply:
okay will remove from editcommand.